tests/test_controls.py

```diff
@@ -20,25 +21,29 @@ def setup_class(self):
                                                  ('resamPars', [0.9, 50]),
                                                  ('display', DisplayOptions.Iter)])
     def test_base_property_values(self, property: str, value: Any) -> None:
+        """Tests the default values of BaseProcedure class."""
         assert getattr(self.base_procedure, property) == value
 
     @pytest.mark.parametrize("property, var_type", [('parallel', ParallelOptions),
                                                     ('calcSldDuringFit', bool),
                                                     ('resamPars', list),
                                                     ('display', DisplayOptions)])
     def test_base_property_types(self, property: str, var_type: type) -> None:
```

Comment:
Isn't this test already covered by the preceeding one? That is to say, there is no way for this test to fail without the previous one also failing. If so, this test should be removed.